modules/ztp-sno-du-configuring-logging-locally-and-forwarding.adoc

```diff
@@ -8,13 +8,13 @@
 
 {sno-caps} clusters that run DU workloads require logging and log forwarding for debugging. The following `ClusterLogging` and `ClusterLogForwarder` custom resources (CRs) are required.
 
-.Recommended cluster logging and log forwarding configuration
+.Recommended cluster logging configuration (ClusterLogging.yaml)
```

Comment:
Since this is technically a heading, I follow this guidance: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings

"Do not use backticks or other markup in assembly or module headings."

Maybe worth getting clarity on this. I will raise a Q on slack.

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

Actually we do have guidance on this.

Reply:
Made your suggested changes :)